tests/cli/test_cli_query.py

```diff
@@ -291,54 +293,81 @@ def test_queries_matching_name_pattern(self, runner):
             ) as f:
                 f.write("SELECT 1")
 
+            os.makedirs("sql/moz-fx-data-test-project")
+            os.mkdir("sql/moz-fx-data-test-project/telemetry_derived")
+            os.mkdir("sql/moz-fx-data-test-project/telemetry_derived/query_v1")
```

Comment:
```suggestion
            os.mkdirs("sql/moz-fx-data-test-project/telemetry_derived/query_v1")
```

Can this be a single `mkdirs` call?